pkgs/development/web/deno/default.nix

```diff
@@ -0,0 +1,106 @@
+{ stdenv, fetchurl, fetchFromGitHub, rust, rustPlatform, python27, installShellFiles }:
```

Comment:
Put these on separate lines - `nixpkgs-fmt` format will show you how.

Reply:
I've ran `nixpkgs-fmt` but it didn't move them automatically. I've manually moved them & after a rerun they're still on their separate lines